chia/data_layer/data_store.py

```diff
@@ -182,46 +201,6 @@ async def check(self) -> None:
         for check in self._checks:
```

Comment:
looks like we only use this in conftest, and if i understand correctly conftest is a bunch of fixtures for the tests, why not move this whole thing to the tests section ?

Reply:
Yes, `conftest.py` is used by pytest.  I still think runtime integrity checks are a valid thing but the only relationship to this PR is that some of the checks are made much less relevant since what they check is now enforced at the schema level.  I guess maybe they should still be part of a `--extra-rigorous` check option but sure, since these aren't exposed via RPC or CLI at this point they could be moved to the tests where they are used.  They could move back if we end up exposing them.